bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/mascot/MascotCat.scala

```diff
@@ -92,17 +92,17 @@ object MascotCat {
    * @param centerRA the base position RA coordinate
    * @param centerDec the base position Dec coordinate
    * @param band determines which magnitudes are used in the calculations: (one of "B", "V", "R", "J", "H", "K")
-   * @param progressCheck called for each asterism as it is calculated, can cancel the calculations by returning false
+   * @param shouldContinue called for each asterism as it is calculated, can cancel the calculations by returning false
    * @return a tuple: (list of stars actually used, list of asterisms found)
    */
   def findBestAsterismInTargetsList(javaList: List[SiderealTarget],
                                     centerRA: Double, centerDec: Double,
                                     band: BandsList, factor: Double,
-                                    progressCheck: (Strehl, Boolean) => Boolean): StrehlResults = {
+                                    shouldContinue: (Strehl, Boolean) => Boolean): StrehlResults = {
     val progress:ProgressFunction = (s: Strehl, count: Int, total: Int) => {
       defaultProgress(s, count, total)
-      // Why usable defaults to true?
-      progressCheck(s, true)
+      // TODO: Why is the usable parameter always true?
+      shouldContinue(s, true)
```

Comment:
Incidentally, the meaning of `true` and `false` never did change, except in the comments for one method for added confusion, as the method `shouldContinue` was passed to and used promptly reverted back to the intended meaning.

I maintained the original functionality (mostly) and negated.

New commit in a second.